stackslib/src/chainstate/stacks/miner.rs

```diff
@@ -2222,10 +2222,22 @@ impl StacksBlockBuilder {
         let mempool_settings = settings.mempool_settings.clone();
         let ts_start = get_epoch_time_ms();
         let stacks_epoch_id = epoch_tx.get_epoch();
-        let block_limit = epoch_tx
+        let remaining_limit = epoch_tx
             .block_limit()
             .expect("Failed to obtain block limit from miner's block connection");
 
+        // Attempt to spread the cost limit across the tenure by always using only 25% of the remaining budget
```

Comment:
Do we want this to be configurable?

Also, do we want to multiply and then divide?

Reply:
I didn't multiply and then divide because I figured it would be more likely to overflow than to underflow. will only ever underflow if its less than 100.